### PR TITLE
Added a page on _tasks API

### PR DESCRIPTION
**Pull request by keithhc2**
Issue #394 

*Description of changes:*
Added a page on `_tasks API` that covers available parameters, cancelling tasks, etc.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/elasticsearch/tasksapis.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
+layout: default
+title: Tasks API
+parent: Elasticsearch
+nav_order:  8
+has_math: false
+---
+
+# Tasks API
```
Comment (aetter) [inline docs/elasticsearch/tasksapis.md:9]: Hey @keithhc2, the introduction here doesn't need to be substantially longer, but it does need to tell people what this feature is for. What is a task in this context? What are some example tasks that might take a while? Are certain calls to the cluster automatically considered tasks and added to the list, or do I have to add any arguments to my calls?

File: docs/elasticsearch/tasksapis.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+                  "parent_task_id": "Mgqdm0r9SEGClWxp_RbnaQ:24578",
+                  "headers": {}
+              }
+          }
+      }
+  }
+}
+```
+
+## Task Cancelling
```
Comment (aetter) [inline docs/elasticsearch/tasksapis.md:83]: Lowercase for all headers, so "Task canceling"

File: docs/elasticsearch/tasksapis.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+You can use the `_tasks` API to see what tasks are currently executing in your cluster.
+
+This request returns information about all of your tasks:
+
+```
+GET _tasks
+```
+
+By including a task ID, you can get information specific to a particular task. Note that a task's ID consists of a node's identifying string and the task's numerical ID. For example, if your node's identifying string is `nodestring` and the task's numerical ID is `1234`, then your task's ID is `nodestring:1234`. You can find this information by running the `_tasks` API.
```
Comment (aetter) [inline docs/elasticsearch/tasksapis.md:19]: Note that a task ID consists of

File: docs/elasticsearch/tasksapis.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+              }
+          }
+      }
+  }
+}
+```
+
+## Task Cancelling
+
+After getting a list of tasks, you can cancel all cancellable tasks with this request:
```
Comment (aetter) [inline docs/elasticsearch/tasksapis.md:85]: It's not a big deal, but our editor tells me it's: cancel, canceled, canceling, and cancellation. Following this logic, I believe we want cancelable.

File: docs/elasticsearch/tasksapis.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+POST _tasks/_cancel
+```
+
+You can also cancel a task by including specific task ID.
+
+```
+POST _tasks/<task_id>/_cancel
+```
+
+The cancel operation supports the same parameters as the `tasks` API, so you can cancel all cancellable tasks on multiple nodes like in the following example:
```
Comment (aetter) [inline docs/elasticsearch/tasksapis.md:97]: so you can also cancel all tasks on multiple nodes:

File: docs/elasticsearch/tasksapis.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+          }
+        }
+      }
+    }
+  }
+}
+```
+This operation supports the same parameters as the `tasks` API, so you can associate `X-Opaque-Id` with specific tasks.
+
+This command associates `X-Opaque-Id: 123456` with tasks in the node `odfe-node1`.
```
Comment (aetter) [inline docs/elasticsearch/tasksapis.md:167]: Remove line, add colon to previous paragraph.

**Review by aetter — CHANGES_REQUESTED** [review]

File: docs/elasticsearch/tasksapis.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+parent_task_id | string | Return tasks with specified parent task id (node_id:task_number). Leave empty or set to -1 to return all.
+wait_for_completion | boolean | Wait for the matching tasks to complete. (default: false)
+group_by | enum | Group tasks by parent/child relationships or nodes. (default: nodes)
+timeout | time | Explicit operation timeout. (default: 30 seconds)
+master_timeout | time | Time to wait for a connection to the master node. (default: 30 seconds)
+
+For example, this sample request returns tasks currently running on a node named `odfe-node1`.
+
+```json
+GET /_tasks?nodes=odfe-node1
```
Comment (ashwinkumar12345) [inline docs/elasticsearch/tasksapis.md:41]: @keithhc2 - Maybe we add 'Sample Response' here to make the distinction between request and response clear.

File: docs/elasticsearch/tasksapis.md (hunk 4 above)
Comment (ashwinkumar12345) [reply docs/elasticsearch/tasksapis.md:85]: Again not a big deal, some context for what makes a task 'cancelable' might be useful.

File: docs/elasticsearch/tasksapis.md (hunk 5 above)
Comment (ashwinkumar12345) [reply docs/elasticsearch/tasksapis.md:97]: Are the two aspects of the sentence related? If not, maybe break it up into a separate sentence 'To cancel all tasks on multiple nodes:'?

**Review by ashwinkumar12345 — APPROVED** [review]

**Pull request closed without merging**